### ISSUE TITLE
Various MPS documents contain syntax and reference errors

### ISSUE THREAD

**Issue by rptb1** [Ravenbrook/mps#113]
Various minor errors have crept into MPS documents, such as design documents.

I noticed some by inspection then wrote a command line to run documents through rst2html and found these:

```
./procedure/release-build.rst:283: (WARNING/2) malformed hyperlink target.
./procedure/release-build.rst:276: (ERROR/3) Unknown target name: "pnj".
./design/protix.txt:4: (WARNING/2) Duplicate explicit target name: ".fun.set.assume.mprotect".
./design/config.txt:4: (WARNING/2) Duplicate explicit target name: ".var".
./design/arena.txt:4: (WARNING/2) Duplicate explicit target name: ".tract.field.base".
./design/poolawl.txt:247: (ERROR/3) Unknown target name: ".poolstruct.gen".
./design/poolamc.txt:4: (WARNING/2) Duplicate explicit target name: ".intro".
./design/poolamc.txt:4: (WARNING/2) Duplicate explicit target name: ".readership".
./design/shield.txt:259: (ERROR/3) Unknown target name: ".inv.unsynced.suspend".
./design/prmc.txt:4: (WARNING/2) Duplicate explicit target name: ".if.init.thread".
./design/protocol.txt:572: (ERROR/3) Unknown target name: ".if.subclass".
```

Not hard to fix up.  Care must be taken not to change tags without searching for references.  Grep is your friend.

We can use some automation to prevent them happening again.

**Comment by UNAA008** [comment]
Submission of #112 caused errors in reStructedText files.
https://github.com/Ravenbrook/mps/actions/runs/4146422245/jobs/7172058645

**Comment by rptb1** [comment]
> Submission of #112 caused errors in reStructedText files. https://github.com/Ravenbrook/mps/actions/runs/4146422245/jobs/7172058645

Fixed by #141 in https://github.com/Ravenbrook/mps/commit/eed418b5477ce651b062a4f97424fdfc4aa4e725

**Comment by rptb1** [comment]
> That indicates a flaw in the [proc.merge.pull-request](https://github.com/Ravenbrook/mps/blob/40f23128c2bcddd17aadc0c4c3369e6473c754a1/procedure/pull-request-merge.rst#5-merging-procedure) checklist item 9, because the trial merge wasn't reliable, being quite old. A catch-up merge from master would've kicked the CI and found this before the commit to master. Or we could kick the CI with tools.

Fixed by #152 in https://github.com/Ravenbrook/mps/commit/990612d7652315cf325318a46f18e78725ff29de
